scripts/consolidate-logs_test-concurrent-agents.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
Could you remove the `agent-log-1.txt` and `agent-log-2.txt` files? Guess it was created as part of some script you ran?

Reply:
Good catch. Thanks.

Reply:
Done